doc/manual/src/release-notes/rl-next.md

```diff
@@ -39,3 +39,8 @@
 
   * `man nix-store-<operation>` and `man nix-env-<operation>`
   * `nix-store --help --<operation>` and `nix-env --help --<operation>`.
+
+* Nix now checks whether the the store trusts you, the client.
+  This is useful for scripting interactions with (non-legacy-ssh) remote Nix stores.
```

Comment:
This message feels a bit weird. Nix obviously already checked that. There just wasn't a way to get this information from the client side